build.rs

```diff
@@ -0,0 +1,27 @@
+use std::env;
+
+fn main() {
+    if let Ok(version) = env::var("DEP_CARES_VERSION_NUMBER") {
+        let version = u64::from_str_radix(&version, 16).unwrap();
+
+        if version >= 0x1_0f_00 {
+            // 1.15.0
+            println!("cargo:rustc-cfg=cares115");
```

Comment:
these want some sort of delimiter eg `cares1_15`

it certainly doesn't look likely to happen soon, but perhaps one day there'll be a c-ares 11.5...